From 183a0edb25b205c96f8f790816badcd2894ccb1f Mon Sep 17 00:00:00 2001
From: Xu Jianqun <jay.xu@rock-chips.com>
Date: Tue, 5 Jan 2016 19:32:49 +0800
Subject: [PATCH] ARM64: configs: rockchip_defconfig select goodix driver

Select goodix driver for gt911, used by rk3368 Sheep board.

Change-Id: Ic4c6fa8e4fa4e54ab89e6536b02fffa6d07cecd7
Signed-off-by: Xu Jianqun <jay.xu@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 0a15e905679f..539ba2a52537 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -258,6 +258,7 @@ CONFIG_TABLET_USB_GTCO=y
 CONFIG_TABLET_USB_HANWANG=y
 CONFIG_TABLET_USB_KBTAB=y
 CONFIG_INPUT_TOUCHSCREEN=y
+CONFIG_TOUCHSCREEN_GOODIX=y
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_UINPUT=y
 # CONFIG_SERIO is not set
-- 
2.35.3

